config/tcs_config.toml

```diff
@@ -50,11 +51,13 @@ ias_url = "https://api.trustedservices.intel.com/sgx/dev"
 # Proxy for https. Leave commented out for direct Internet connections or
 # uncomment and change to your corporate proxy.
 #https_proxy = "http://your-proxy:your-port/"
+https_proxy = "http://proxy-chain.intel.com:912"
```

Comment:
remove hard-coding intel proxies

Reply:
Do not use an internal Intel proxy value here.  Keep the original example.

Reply:
Ok